packages/discord.js/src/structures/Webhook.js

```diff
@@ -393,6 +403,22 @@ class Webhook {
     return this.avatar && this.client.rest.cdn.Avatar(this.id, this.avatar, options);
   }
 
+  /**
+   * Whether this webhook is created by a user.
+   * @returns {boolean}
+   */
+  isUserCreated() {
+    return Boolean(this.owner);
+  }
+
+  /**
+   * Whether this webhook is created by an application and not a user.
```

Comment:
I would remove the "and not a user" portion of this string, it's not needed information.
```suggestion
   * Whether this webhook is created by an application.
```